lib/bson/decimal128.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
-var Long = require('./long');
+var Long = require('./long'),
+  f = require('util').format;
```

Comment:
as long as you're already here, the preferred type for top level includes is `const` since they are not meant to change.